content/ri/installing/system-requirements.md

```diff
@@ -0,0 +1,37 @@
+---
+Title: System requirements for RedisInsight
+date: 2021-01-29 03:49:29 +0530
+weight: 4
+categories: ["RI"]
+path: install/system-requirements/
+---
+This section of the documentation describes the recommended requirements for various Operating Systems for running RedisInsight.
+
+## Windows
+
+- Windows 10
+- 8GB Memory (RAM)
+- 32 bit or 64 bit (Application is 32 bit so it will work on both)
+- No software pre-requisites. RI should install and run on a fresh Windows system, no need to install any .NET framework or anything like that.
+
+## Mac
+
+- Max OS X 10.9 or later
```

Comment:
I think this should be 10.13 or later. See [here](https://docs.redislabs.com/latest/ri/installing/install-redis-desktop/).

Reply:
@nishantgeorge these are the requirments for the desktop client, correct? Would it make more sense for them to be included in the installation instructions?

Reply:
@kaitlynmichaelredislabs We have linked the recommended system requirements from within the installation instructions. 
The intention is to not overcrowd the installation instructions with other information.